doc/source/parameters/cfd/physical_properties.rst

```diff
@@ -481,8 +488,27 @@ This model is parameterized using the following section:
 
 * The ``specific heat solid`` is :math:`C_{p,s}`
 
-Cahn-Hilliard Mobility Model
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Interface Physical Property Models
+***********************************
+
+.. _surface_tension_models:
+
+Surface Tension Models
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Lethe supports two types of surface tension models: ``constant`` and ``linear``. A ``constant`` surface tension model assumes a constant value of surface tension. While a ``linear`` surface tension assumes that the surface tension evolves linearly with the temperature:
+
+.. math::
+  \sigma(T) = \sigma_0 + \frac{d\sigma}{dT} \cdot T
```

Comment:
I think the most common way to write this is:

```suggestion
  \sigma(T) = \sigma_0 + \frac{d\sigma}{dT} \cdot (T-T_0)
```
where :math:`sigma_0` and :math:`\frac{d\sigma}{dT}`  are the surface tension coefficient and its dependence to the temperature at  :math:`T_0`.

I don't think that's what is currently implemented in Lethe :o 

We can also keep it like this, but specify that \sigma_0 is the value at T=0.

Reply:
I'll add the specification and we could review the formulation in the code in a future PR! :)